slam_toolbox/package.xml

```diff
@@ -37,34 +39,37 @@
   <build_depend>libqt5-widgets</build_depend>
   <build_depend>qtbase5-dev</build_depend>
   <build_depend>map_server</build_depend>
-  <run_depend>slam_toolbox_msgs</run_depend>
-  <run_depend>eigen</run_depend>
-  <run_depend>pluginlib</run_depend>
-  <run_depend>message_filters</run_depend>
-  <run_depend>nav_msgs</run_depend>
-  <run_depend>rosconsole</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>sparse_bundle_adjustment</run_depend>
-  <run_depend>sensor_msgs</run_depend>
-  <run_depend>tf2</run_depend>
-  <run_depend>tf2_ros</run_depend>
-  <run_depend>visualization_msgs</run_depend>
-  <run_depend>message_runtime</run_depend>
-  <run_depend>std_srvs</run_depend>
-  <run_depend>interactive_markers</run_depend>
-  <run_depend>std_msgs</run_depend>
-  <run_depend>suitesparse</run_depend>
-  <run_depend>rviz</run_depend>
-  <run_depend>liblapack-dev</run_depend>
-  <run_depend>libceres-dev</run_depend>
-  <run_depend>libg2o</run_depend>
-  <run_depend>tf2_geometry_msgs</run_depend>
-  <run_depend>tbb</run_depend>
-  <run_depend>libqt5-core</run_depend>
-  <run_depend>libqt5-widgets</run_depend>
-  <run_depend>map_server</run_depend>
+  <exec_depend>slam_toolbox_msgs</exec_depend>
+  <exec_depend>eigen</exec_depend>
+  <exec_depend>pluginlib</exec_depend>
+  <exec_depend>message_filters</exec_depend>
+  <exec_depend>nav_msgs</exec_depend>
+  <exec_depend>rosconsole</exec_depend>
+  <exec_depend>roscpp</exec_depend>
+  <exec_depend>sparse_bundle_adjustment</exec_depend>
+  <exec_depend>sensor_msgs</exec_depend>
+  <exec_depend>tf2</exec_depend>
+  <exec_depend>tf2_ros</exec_depend>
+  <exec_depend>visualization_msgs</exec_depend>
+  <exec_depend>message_runtime</exec_depend>
+  <exec_depend>std_srvs</exec_depend>
+  <exec_depend>interactive_markers</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
+  <exec_depend>suitesparse</exec_depend>
+  <exec_depend>rviz</exec_depend>
+  <exec_depend>liblapack-dev</exec_depend>
+  <exec_depend>libceres-dev</exec_depend>
+  <exec_depend>libg2o</exec_depend>
+  <exec_depend>tf2_geometry_msgs</exec_depend>
+  <exec_depend>tbb</exec_depend>
+  <exec_depend>libqt5-core</exec_depend>
+  <exec_depend>libqt5-widgets</exec_depend>
+  <exec_depend>map_server</exec_depend>
 
-  <test_depend>gtest</test_depend>
+  <depend>rostest</depend>
```

Comment:
This doesn't seem right... Shouldn't the test_depend be enough?

Reply:
apparently not. I'm trying to figure it out